suite-native/passphrase/src/useFeatureRequestsPassphrase.tsx

```diff
@@ -0,0 +1,68 @@
+import { useCallback, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+
+import TrezorConnect from '@trezor/connect';
+import {
+    PassphraseStackParamList,
+    PassphraseStackRoutes,
+    RootStackParamList,
+    RootStackRoutes,
+    StackToStackCompositeNavigationProps,
+} from '@suite-native/navigation';
+import {
+    deviceActions,
+    selectDevice,
+    selectDeviceButtonRequestsCodes,
+} from '@suite-common/wallet-core';
+
+type NavigationProp = StackToStackCompositeNavigationProps<
```

Comment:
Will theme navigation props work for any screen? Now it works, because it is only used on a `ReceiveModalScreen` that is placed in RootStack. Not sure if it would work for example for navigation from `ReceiveStack` or `SendStack`. Could you verify that, please? If it is not gonna work, then the navigation handler should be passed to this hook as an argument.

Reply:
These are just typings, but the navigation works the same across the whole app. Wherever you call `navigation.goBack()`, it will do that as long as navigation is available and there is a screen to go back to. Since it's currently used primarily on the RootStack, it's typed that way and should there be need to extend this type, we can do `RootStack | AnyOtherStack`, but since all other screens, that are not directly in RootStack are subset of RootStack, the typing should be future proof. Good thought though

Reply:
Btw. If `navigation.goBack` wouldn't work, it throws runtime error so then it's pretty clear what failed and can be debugged. But it's same as everywhere in our navigation logic